app/views/content/salaries-and-benefits.md

```diff
@@ -130,29 +125,17 @@ The salary ranges for headteachers are:
 | Outer London                             | £57,124 | £134,765 |
 | Inner London                             | £62,304 | £139,891 |
 
-### Other payments
+## If you do not have qualified teacher status (QTS)
```

Comment:
@gemmadallmandfe @philspray created a new h2 for this section as it just seemed to sit really oddly alongside the other bits